### ISSUE TITLE
`CKV_AZURE_141` cannot detect default value when aks resource is defined in a module.

### ISSUE THREAD

**Issue by lonegunmanb** [bridgecrewio/checkov#3193]
**Describe the issue**
`CKV_AZURE_141`

**Examples**

`main.tf` in the `sub` folder:

```hcl
variable "local_account_disabled" {
  default = null
}

resource "azurerm_resource_group" "example" {
  name     = "example-resources"
  location = "West Europe"
}

resource "azurerm_kubernetes_cluster" "example" {
  name                = "example-aks1"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  dns_prefix          = "exampleaks1"
  local_account_disabled = var.local_account_disabled

  default_node_pool {
    name       = "default"
    node_count = 1
    vm_size    = "Standard_D2_v2"
  }

  identity {
    type = "SystemAssigned"
  }

  tags = {
    Environment = "Production"
  }
}
```

`main.tf` in the root folder:

```hcl
module "aks" {
  source = "./sub"
}
```

**Version (please complete the following information):**
 - 2.1.36

**Additional context**
The check works well when I set `var.local_account_disabled` explicitly. The only failed case is when I use the default value, and according to the [document](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#local_account_disabled), the default value is `false` and should be detected by Checkov.

Checkov can detect default value when I defined the aks resource in the root module.
